yucca/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+
+services:
+  yucca:
+    image: yuccastream/yucca:0.10.0
+    restart: on-failure
+    shm_size: "128mb"
+    volumes:
+      - "${APP_DATA_DIR}/data/data:/opt/yucca/data"
+      - "${APP_DATA_DIR}/data/ffmpeg:/opt/yucca/ffmpeg"
+    ports:
+      - 9910:9910 # Web UI
```

Comment:
It would probably be better to serve the web ui through app-proxy. Like in this guide: https://github.com/getumbrel/umbrel-apps?tab=readme-ov-file#2-%EF%B8%8Fpackaging-the-app-for-umbrel

Reply:
done